web/client/themes/default/ms-variables.less

```diff
@@ -96,6 +97,7 @@
     // **************
     // color for the base page and components
     main-bg: --ms-main-bg, @ms-main-bg;
+    mask-bg: --ms-mask-bg, @ms-mask-bg;
```

Comment:
This variable is not used please remove it

Reply:
it's used in longitudinal-profile.less 

![image](https://github.com/geosolutions-it/MapStore2/assets/11991428/07eb10fb-c718-4c61-b854-126e7258c050)